# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/media-conduit/VideoConduit.h
# Commit: 551533329f3f
# Full Hash: 551533329f3fdb48cb2df554f2fbbb4d6ce1f877
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-09 10:01:40
# Regressor Bug: 1376873
# File Overlap Count: 3
# Description:
#   Bug 1376873 - Use audio/video sync groups; r=bwc
#   
#   This uses sync groups in the receive stream configs for the conduits rather
#   than establishing sync through direct calls. When the streams are created
#   in call.cc, ConfigureSync is called, which results in SetSync being called
# ==============================================================================

diff -r b55b0368d9f2 -r 551533329f3f media/webrtc/signaling/src/media-conduit/VideoConduit.h
--- a/media/webrtc/signaling/src/media-conduit/VideoConduit.h	Wed Feb 07 15:00:17 2018 -0500
+++ b/media/webrtc/signaling/src/media-conduit/VideoConduit.h	Fri Apr 27 07:25:19 2018 -0400
@@ -84,11 +84,6 @@
                         const RtpExtList& aExtensions) override;
 
   /**
-   * Set up A/V sync between this (incoming) VideoConduit and an audio conduit.
-   */
-  void SyncTo(WebrtcAudioConduit *aConduit);
-
-  /**
    * Function to attach Renderer end-point for the Media-Video conduit.
    * @param aRenderer : Reference to the concrete mozilla Video renderer implementation
    * Note: Multiple invocations of this API shall remove an existing renderer
@@ -250,6 +245,8 @@
   bool SetLocalCNAME(const char* cname) override;
   bool SetLocalMID(const std::string& mid) override;
 
+  void SetSyncGroup(const std::string& group) override;
+
   bool GetRemoteSSRCLocked(unsigned int* ssrc);
   bool SetRemoteSSRCLocked(unsigned int ssrc);
 